### PR TITLE
 deploy full d2cc process

### PR DESCRIPTION
**Pull request by kahyami**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: k8s/cse/d2cc/cse-d2cc-gridcapa-runner-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        username: ${RABBITMQ_USER}
+        password: ${RABBITMQ_PASSWORD}
+        port: 5672
+        template:
+          reply-timeout: 7200000
+      cloud:
+        stream:
+          bindings:
+            task-status-update:
+              destination: cse-idcc-task-status-updated
```
Comment (TheMaskedTurtle) [inline k8s/cse/d2cc/cse-d2cc-gridcapa-runner-configmap.yaml:24]: Change to d2cc

File: k8s/cse/d2cc/cse-d2cc-gridcapa-runner-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    logback:
+      amqp:
+        exchange: cse-idcc-task-manager-logs-exchange
```
Comment (TheMaskedTurtle) [inline k8s/cse/d2cc/cse-d2cc-gridcapa-runner-configmap.yaml:28]: Change to d2cc

File: k8s/cse/d2cc/cse-d2cc-gridcapa-runner-configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+          node-fr: FFG.IL14
+          node-it: FFG.IL13
+          fictive-lines:
+            - FFG.IL14 FFG.IL13 1
+            - FFG.IL14 FFG.IL13 2
+            - FFG.IL14 FFG.IL13 3
+      bindings:
+        request:
+          destination: cse-request
+          group: cse-idcc-runner
```
Comment (TheMaskedTurtle) [inline k8s/cse/d2cc/cse-d2cc-gridcapa-runner-configmap.yaml:73]: Change to d2cc

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: docker-compose/config/cse/d2cc/cse-d2cc-gridcapa-job-launcher.yml [hunk 4]
```diff
@@ -14,16 +14,17 @@ spring:
     stream:
       bindings:
         run-task:
-          destination: cse-d2cc-run-task
+          destination: cse-d2cc-launch-task
           group: cse-d2cc-job-launcher
 
 job-launcher:
   url:
     task-manager-timestamp-url: http://cse-d2cc-gridcapa-task-manager:8080/tasks/
```
Comment (TheMaskedTurtle) [inline docker-compose/config/cse/d2cc/cse-d2cc-gridcapa-job-launcher.yml:22]: the host should match the name of the service in the docker-compose file, here cse-d2cc-task-manager

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: k8s/overlays/azure/test/kustomization.yaml [hunk 5]
```diff
@@ -2,7 +2,9 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 bases:
-  - ../../../cse/idcc
+  - ../cse/idcc
+  - ../cse/d2cc
+  - ../cse/common
```
Comment (TheMaskedTurtle) [inline k8s/overlays/azure/test/kustomization.yaml:7]: Wrong paths, should be ../../../cse/idcc or d2cc or common

File: k8s/overlays/azure/dev/kustomization.yaml [hunk 6]
```diff
@@ -2,7 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 bases:
-  - ../../../cse/d2cc
+  - ../cse/d2cc
+  - ../cse/common
```
Comment (TheMaskedTurtle) [inline k8s/overlays/azure/dev/kustomization.yaml:6]: Wrong paths, should be ../../../cse/d2cc or common

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: k8s/cse/d2cc/cse-d2cc-gridcapa-adapter-configmap.yaml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+        host: gridcapa-rabbitmq
+        port: 5672
+        template:
+          reply-timeout: 14400000
+      cloud:
+          stream:
+            bindings:
+              handleRun-in-0:
+                binder: rabbit
+                destination: cse-d2cc-launch-task
```
Comment (TheMaskedTurtle) [inline k8s/cse/d2cc/cse-d2cc-gridcapa-adapter-configmap.yaml:25]: Mismatch between this exchange and the one from job-launcher. There are cse-d2cc-run task and cse-d2cc-launch-task. Must be aligned.

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: docker-compose/standalone-d2cc/docker-compose.yml [hunk 8]
```diff
@@ -249,6 +249,7 @@ services:
     image: farao/gridcapa-cse-adapter:latest
     volumes:
       - ../config/cse/d2cc/cse-d2cc-gridcapa-adapter-application.yml:/config/application.yml
+      - ../config/cse/d2cc/20211209_Targets_CH.xml:/config/target-ch.xml
```
Comment (kahyami) [inline docker-compose/standalone-d2cc/docker-compose.yml:252]: @TheMaskedTurtle  please update the docker-compose file in the acceptance-tests package

Comment (TheMaskedTurtle) [reply docker-compose/standalone-d2cc/docker-compose.yml:252]: done

**Review by TheMaskedTurtle — APPROVED** [review]

**Pull request merged**